src/main/java/io/xconn/cryptobox/SealedBox.java

```diff
@@ -0,0 +1,89 @@
+package io.xconn.cryptobox;
+
+import org.bouncycastle.crypto.digests.Blake2bDigest;
+import org.bouncycastle.crypto.engines.XSalsa20Engine;
+import org.bouncycastle.crypto.macs.Poly1305;
+import org.bouncycastle.crypto.params.KeyParameter;
+import org.bouncycastle.crypto.params.ParametersWithIV;
+import org.bouncycastle.math.ec.rfc7748.X25519;
+import org.bouncycastle.util.Arrays;
+
+import static io.xconn.cryptobox.SecretBox.boxOpen;
```

Comment:
```suggestion
import static io.xconn.cryptobox.SecretBox;
```

We should import SecretBox and call any functions on it for clarity.